### PR TITLE
Added cursor pointer to the sidebar folder

### PR DESCRIPTION
**Pull request by rkgupta21**
Fixed issue [#1703 ](https://github.com/mozilla/thimble.mozilla.org/issues/1703) 

Added `cursor: pointer !important;` to `li.jstree-open>a>span ` and `li.jstree-closed>a>span ` css file. 
screen shot 2017-02-09 at 12 33 35 pm

### CODE REVIEW

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css [hunk 1]
```diff
@@ -25,10 +25,12 @@ li.jstree-leaf>a>span {
 li.jstree-open>a>span {
 	font-size: 15px;
 	color: #666666;
+	cursor: pointer !important;
 }
 li.jstree-closed>a>span {
 	font-size: 15px;
 	color: #666666;
+        cursor: pointer !important;
```
Comment (gideonthomas) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: there seems to be an extra space before the word `cursor`

**Comment by flukeout** [comment]
Checking now.

**Comment by flukeout** [comment]
Hi @rkgupta21 thank you for the PR. It works mostly as expected, except for two changes:

* The entire row containing the folder name is clickable, but the pointer cursor only appears when you hover over the name of the folder.
* I think we can simplify the CSS into just the rule below...

```
.jstree-brackets li a:hover {
  cursor: pointer;
}
```
This also helps us avoid the ``!mportant`` rules, which are not needed. I found this selector by inspecting the ``<a>`` element to see where it was getting the ``cursor: default`` style from...

image

...so then I just wrote a rule to undo that.

**Comment by rkgupta21** [comment]
Fixed the cursor position.

**Comment by flukeout** [comment]
Checking it out now, thanks.

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css [hunk 2]
```diff
@@ -24,7 +24,7 @@ li.jstree-leaf>a>span {
 }
 li.jstree-open>a>span {
 	font-size: 15px;
-	color: #666666;
+	color: #666666;
```
Comment (flukeout) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: Can you get rid of the trailing space here?

**Comment by flukeout** [comment]
Looks good, tested in Chrome + FF, can you get rid of the extra space and we'll get it merged?

**Comment by gideonthomas** [comment]
Looks like the changes have been made! Thanks for the contribution @rkgupta21!

**Pull request merged**